### PR TITLE
 [W5.7][W10-2]Low Yew Woei(Aiden)

### PR DESCRIPTION
**Pull request by aidenywl**
Request to add Sort Command feature that prints out everything in the addressbook by alphabetical order.

### CODE REVIEW

**Comment by katenhy** [comment]
Nice work! Follow the original syntax and coding standard. 
Sorry for the late comment and really appreciate your help today. Thanks so much!

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -10,6 +10,26 @@
...
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
+    private void assertSomeNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
```
Comment (damithc) [inline test/java/seedu/addressbook/common/UtilsTest.java:32]: In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the `master` branch that is in sync with the upstream `master` branch

**Review by damithc — APPROVED** [review]
:+1: Good effort on this PR @yewwoei. In particular, good job on,
* updating docs

Suggestions to improve further:
* Update I/O tests and JUnit tests
* Limit the PR to one enhancement and avoid including changes not related to that enhancement or unmerged  code from other PRs

**Comment by damithc** [comment]
As before, please close the PR after viewing comments.

**Pull request closed without merging**